javascript/javascript3/week2/week2.js

```diff
@@ -0,0 +1,59 @@
+//Create an array of bad movies
+
+const moviesURL = 'https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json';
+fetch(moviesURL)
+.then(response => response.json())
+.then(data => {
+    const badMovies = data.filter(movie => movie.rating <= 5);
+    badMovies.forEach(element => {
+        element.title;
+        console.log(element.title);
+    })
+});
+
+
+//Create an array of bad movies since year 2000
+
+//const moviesURL = 'https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json';
+fetch(moviesURL)
+.then(response => response.json())
+.then(data => {
+    const badMovies = data.filter(movie => (movie.rating <= 5));
+    const badMoviesAfterYear =  badMovies.filter (movie => (movie.year = 2000));
+    badMoviesAfterYear.forEach(element => {
+        element.title;
+        console.log(element.title);
+    })
+}); 
```

Comment:
You can make the two arrays by only one call to the api instead of to calls.

For example:

```suggestion
fetch(moviesURL)
.then(response => response.json())
.then(data => {
    const badMovies = data.filter(movie => movie.rating <= 5);
    const badMoviesSince2000 = badMovies.filter(movie => movie.year >= 2000)
}); 
```

This way you create the two arrays with only one call.